src/App.tsx

```diff
@@ -1,35 +1,41 @@
 import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
+import SearchBar from './components/SearchBar/SearchBar';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state = {
+    query: '',
+  };
+
+  changeState = (value: string) => {
```

Comment:
```suggestion
  setQuery = (value: string) => {
```
changeState слишком обобщенное название